docs/teams/README.md

```diff
@@ -0,0 +1,15 @@
+---
+original:
+  authors: Fiona Gillan, Richard Barlow, Rob Gilton and Thomas Leese
+  url: https://github.com/fgillan/sr-team-coord/wiki/Home
+---
+Welcome to the sr-team-coord wiki which contains the relevant information for the Team Coordinator role.
+
+| Local Team Coordinator | Area covered |
```

Comment:
I think we can remove this table entirely

Reply:
9b2973a rewrites the pae.